src/moduler/verktoylinje/navigasjonslinje.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PT from 'prop-types';
+import { connect } from 'react-redux';
+import { selectDialoger } from '../dialog/dialog-selector';
+import { dialogFilter } from '../filtrering/filter/filter-utils';
+import { LenkepanelBase } from 'nav-frontend-lenkepanel/lib';
+import { Panel } from 'nav-frontend-paneler';
+import { selectErVeileder } from '../identitet/identitet-selector';
+import DialogIkon from '../aktivitet/visning/underelement-for-aktivitet/dialog/DialogIkon';
+
+const DITTNAV_PATH = '/dittnav/';
+const DIALOG_PATH = '/arbeidsrettet-dialog';
+
+class Navigasjonslinje extends Component {
+    render() {
+        const { erVeileder, antallUlesteDialoger } = this.props;
+        if (erVeileder) {
+            return null;
+        } else {
+            return (
+                <Panel className="navigasjonslinje">
+                    <LenkepanelBase className="tilDittNav" href={DITTNAV_PATH}>
+                        <span className="tekst">Ditt NAV</span>
+                    </LenkepanelBase>
+                    <LenkepanelBase className="tilDialog" href={DIALOG_PATH}>
+                        <DialogIkon antallUleste={antallUlesteDialoger} />
+                        <span className="avstand" hidden={antallUlesteDialoger > 0} />
+                        <span className="tekst">Dialog</span>
```

Comment:
`tekst` er nok et for generelt css klassenavn. Er nok flere steder tekst fremkommer fra decoratøren f.eks så er nok lurt at denne blir mer unikt. Eks: navigasjonslinje__tekst
Samme med `avstand`

Reply:
Bruker kun `tekst` som et nested element, dvs. i konteksten `navigasjonslinje.tilDialog` og da mener jeg det er innafor.

Ville selvsagt aldri brukt `tekst` og bare det.

Reply:
men andre vei så kan vi få issues. Hvis decoratøren, eller andre apps har satt style på alt som tilhører `tekst`. Så tenker fremdeles det blir for generell